pip/src/interpreter.rs

```diff
@@ -27,11 +31,33 @@ use qsc::{
     target::Profile,
     LanguageFeatures, PackageType, SourceMap,
 };
+use qsc_qasm3::{OutputSemantics, ProgramType};
 use resource_estimator::{self as re, estimate_expr};
 use std::{cell::RefCell, fmt::Write, path::PathBuf, rc::Rc};
 
+/// If the classes are not Send, the Python interpreter
+/// will not be able to use them in a separate thread.
```

Comment:
Where did this threading requirement come from?

Reply:
Working with Qiskit backends. For anything that would be run in the interactive under the shared interpreter, everything would be sync.